content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -474,6 +474,56 @@ can lose track of volume labels after restart. This deficiency has been fixed
 in Kubernetes 1.26.
 {{< /note >}}
 
+## Changing the permissions of your's '/proc'
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
```

Comment:
Even better:
```suggestion
{{< feature-state feature_gate_name="ProcMountType" >}}
```

Reply:
@haircommander I don't want to add unnecessary noise, but I thought [KEP-4265](https://github.com/kubernetes/enhancements/issues/4265) is targeting `beta`. Am I missing something over here? Are these the missing docs for `alpha`? Where are the docs for `beta`?

Reply:
Oh, I see. The alpha feature was merged quite some time ago, but there weren't any docs for the feature. This PR just simply adds docs for that feature gate.